### DISCUSSION TITLE
Multiplayer Editing

### DISCUSSION THREAD

**Discussion by Zabrid-Chan** [ppy/osu#15725] [Ideas] [answered]
Make the editor multiplayer so multiple ppl can map the same map at the same time

**Comment by bdach** [comment, marked as answer]
the amount of work required to make collaborative editing work is in months to years. collaborative editors are not easy to get right. this would obviously be a cool feature but it's not something you slap on in 10 minutes.

**Reply by smoogipoo** [reply]
Adding to this, it's something we've definitely discussed having at some point.
